src/styles/nav.scss

```diff
@@ -0,0 +1,50 @@
+.nav {
+  display: flex;
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+    text-transform: uppercase;
+  }
+
+  &__list-el {
+    display: flex;
+
+    &:not(:first-child) {
+      margin-left: 20px;
+    }
+  }
+
+  &__list-link {
+    line-height: 60px;
+    text-decoration: none;
+    color: var(--main-letters-color);
+    position: relative;
+    font-weight: 500;
+    transition-duration: 300ms;
+
+    &:hover {
+      transition-duration: 300ms;
+      color: var(--button-color);
+    }
+  }
+}
+  
+.is-active {
```

Comment:
The .is-active class is not nested within .nav. This could lead to conflicts if .is-active is used in other parts of the application. To avoid potential conflicts, always nest modifier classes within the block or element they modify.